modules/meteroid/crates/meteroid-migrations/refinery/migrations/V0043__onboarding.sql

```diff
@@ -0,0 +1,54 @@
+drop table invoicing_config;
+CREATE TABLE invoicing_entity
+(
+    id                      UUID PRIMARY KEY,
+    local_id                text        NOT NULL,
+    is_default              BOOLEAN     NOT NULL,
+    legal_name              text        NOT NULL,
+    invoice_number_pattern  text        NOT NULL,
+    next_invoice_number     bigint      NOT NULL,
+    next_credit_note_number bigint      NOT NULL,
+    grace_period_hours      integer     NOT NULL,
+    net_terms               integer     NOT NULL,
+    invoice_footer_info     TEXT,
+    invoice_footer_legal    TEXT,
+    logo_attachment_id      text,
+    brand_color             text,
+    address_line1           text,
+    address_line2           text,
+    zip_code                VARCHAR(50),
+    state                   text,
+    city                    text,
+    vat_number              text,
+    country                 text        NOT NULL,
+    accounting_currency     VARCHAR(50) NOT NULL,
+    tenant_id               UUID        NOT NULL,
+    CONSTRAINT "invoicing_entity_tenant_id_fkey" FOREIGN KEY ("tenant_id") REFERENCES "tenant" ("id") ON UPDATE CASCADE ON DELETE CASCADE,
+    CONSTRAINT "invoicing_entity_is_default_tenant_id_key" UNIQUE ("is_default", "tenant_id"),
```

Comment:
shouldn't this unique index applied only when `is_default=true`? the way it is above it restricts to have only 2 entities per tenant